docs/sources/getting-started/glossary.md

```diff
@@ -17,41 +17,72 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
 
 <table>
   <tr>
-    <td style = "vertical-align: top;">Dashboard</td>
-    <td>A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.</td>
+    <td style="vertical-align: top">Dashboard</td>
+    <td>
+      A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Data source</td>
+    <td>
+      A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.
+    </td>
   </tr>
   <tr>
-    <td style = "vertical-align: top;">Data source</td>
-    <td>A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.</td>
+    <td style="vertical-align: top">Exemplar</td>
+    <td>
+      An exemplar is any data that serves as a detailed example of data used to create an aggregation. A common type of examplar is a trace that references an aggregated metric.
```

Comment:
Double "data" here. Also, an exemplar has a mandatory value, which is not mentioned here at all, and an optional timestamp. Proposal:

"An exemplar is any data that serves as a detailed example of one of the observations aggregated into a metric. An exemplar contains the observed value together with an optional timestamp and arbitrary labels, which are typically used to reference a trace."

Reply:
> one of the observations aggregated into a metric

This reads beautifully and clearly! ❤️ 

@joe-elliott suggested "metadata" rather than "data". Is it ok if I change it to:

> An exemplar is metadata that serves as a detailed example of one of the observations aggregated into a metric. An exemplar contains the observed value together with an optional timestamp and arbitrary labels, which are typically used to reference a trace.

Reply:
I would actually avoid the term "metadata". It appears to be more precise, but it might actually create misunderstandings because, in my experience, different groups of people in the metrics space have very different opinions about the line between "data" and "metadata". And exemplars are definitely in that grey area.